Aspen-Discovery/aspen-discovery
# Cookie management db cleanup

This PR clears the user_*_usage tables when the user sets their external search source storage cookies to 0.

TEST PLAN

1. Apply patch
2. Ensure Summon is set up.
3. Ensure Library Cookie Consent is Required:
4. Navigate to the 'Your Cookie Preferences' section in your account. Turn on 'External Search Services' and save.
5. Carry out a Summon Search (the search dropdown should be set to 'Articles & Databases.')
6. Click on a title and then navigate to the user_summon_usage table in the db and note that it has logged your usage.
7. Navigate back to 'Your Cookie Preferences' and turn off the 'External Search Services' and save.
8. Check the db again, update the page and notice that the user_summon_usage table has not changed.
9. In your terminal, run the command adb shell.
10. Navigate to /usr/local/aspen-discovery/code/cron
11. Run the following command: java -jar cron.jar test.localhostaspen
12. Navigate back to the db and refresh the page/.
13. Notice that the entry in user_summon_usage has now been deleted.
